src/ConnectorRuntimeConfig.ts

```diff
@@ -20,7 +20,9 @@ export interface ConnectorRuntimeConfig extends RuntimeConfig {
 
     logging: log4js.Configuration;
 
-    modules: Record<string, ConnectorRuntimeModuleConfiguration>;
+    modules: Record<string, ConnectorRuntimeModuleConfiguration> & {
+        decider: DeciderModuleConfiguration;
```

Comment:
Please keep in mind that we need to add this to our documentation afterwards https://enmeshed.eu/operate/configuration ❤️ @Milena-Czierlinski

Same for the CanCreateRelationship route, which I will add.